datafusion/physical-expr/src/window/nth_value.rs

```diff
@@ -169,9 +164,35 @@ impl PartitionEvaluator for NthValueEvaluator {
         Ok(())
     }
 
-    fn set_state(&mut self, state: &BuiltinWindowState) -> Result<()> {
-        if let BuiltinWindowState::NthValue(nth_value_state) = state {
-            self.state = nth_value_state.clone()
+    /// When the window frame has a fixed beginning (e.g UNBOUNDED
+    /// PRECEDING), some functions such as FIRST_VALUE, LAST_VALUE and
```

Comment:
I think we can replace this line with `PRECEDING), for some functions such as FIRST_VALUE, LAST_VALUE and`

Reply:
Done (BTW if you do "CTRL-G" you can get inline suggestions, to get the github UI to show the suggestion inline):

```suggestion
    /// PRECEDING), for some functions such as FIRST_VALUE, LAST_VALUE and
```

<img width="795" alt="Screenshot 2023-06-12 at 8 11 45 PM" src="https://github.com/apache/arrow-datafusion/assets/490673/12c5c298-e825-4839-b068-cf732f3c09c6">

Reply:
Thanks for the tip